mysite/mysite/settings.py

```diff
@@ -0,0 +1,129 @@
+"""
+Django settings for mysite project.
+
+Generated by 'django-admin startproject' using Django 2.0.7.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.0/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/ 
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '45dteax6vlpaau_ggpqm+46a@mk+nb&ih#i4#_-%y%hw4qty&l'

```

Comment:
Next time you should not include the `SECRET_KEY ` in the public, it's used for many security stuff in Django,
Anyway, this is your first time, but for future project I'll show you how to hide it so you don't expose it to the public like this.